pkg/plugin/javascript.go

```diff
@@ -371,29 +371,39 @@ func (t *jsPlugin) PrintTabs(ctx context.Context, object runtime.Object) ([]TabR
 		return []TabResponse{}, err
 	}
 
-	tab := tabResponse.Get("tab")
-	if tab == goja.Undefined() {
-		return []TabResponse{}, fmt.Errorf("tab property not found")
-	}
-
-	contents, ok := tab.Export().(map[string]interface{})
-	if !ok {
-		return []TabResponse{}, fmt.Errorf("unable to export tab contents")
-	}
-
-	cTab := &component.Tab{}
-	if name, ok := contents["name"]; ok {
-		cTab.Contents = *component.NewFlexLayout(name.(string))
-		cTab.Name = name.(string)
+	responses, ok := tabResponse.Export().([]interface{})
```

Comment:
This works even when the response is a single object? Heh, gotta love JavaScript.